force-app/main/educationCloudSettings/lwc/educationCloudSettings/educationCloudSettings.html

```diff
@@ -1,15 +1,20 @@
 <template>
     <div class="slds-theme_default eda-height_full">
-        <ul class="slds-grid">
-            <li class="slds-col slds-size_4-of-12">
+        <ul class="slds-grid slds-wrap">
+            <li class="slds-col slds-size_12-of-12 slds-var-m-bottom_large">
                 <h2 class="slds-text-heading_medium eda-text_heading_bold slds-var-m-left_small slds-var-m-top_small">
-                    {labelReference.productsTitle}
+                    {labelReference.releaseManagementTitle}
```

Comment:
Note that this label is a working label for now, but one that we can safely release.